extendr-api/src/robj/tests.rs

```diff
@@ -100,13 +100,37 @@ fn test_try_from_robj() {
         assert_eq!(<u16>::try_from(Robj::from(1)), Ok(1));
         assert_eq!(<u32>::try_from(Robj::from(1)), Ok(1));
         assert_eq!(<u64>::try_from(Robj::from(1)), Ok(1));
+        assert_eq!(<usize>::try_from(Robj::from(1)), Ok(1));
         assert_eq!(<i8>::try_from(Robj::from(1)), Ok(1));
         assert_eq!(<i16>::try_from(Robj::from(1)), Ok(1));
         assert_eq!(<i32>::try_from(Robj::from(1)), Ok(1));
         assert_eq!(<i64>::try_from(Robj::from(1)), Ok(1));
+        assert_eq!(<isize>::try_from(Robj::from(1)), Ok(1));
         assert_eq!(<f32>::try_from(Robj::from(1)), Ok(1.));
         assert_eq!(<f64>::try_from(Robj::from(1)), Ok(1.));
 
+        assert_eq!(<std::num::NonZeroU8>::try_from(Robj::from(1)).ok(), std::num::NonZeroU8::new(1));
```

Comment:
Could you refactor these into the new `rstest` cases?

Reply:
I have made it use rstest, but right now I only see the need for one case. Is that good enough?

Reply:
No, I meant one case per assertion. You might have to do some magic with trait objects here.

Reply:
Isn't that just obfuscating at this point? I'll give it a try though..

Reply:
It gives us more granular failures, it more accurately measures how many tests we have, and it lets you reduce some boilerplate (ideally).